proto/prysm/v1alpha1/beacon_state.proto

```diff
@@ -289,6 +289,59 @@ message BeaconStateCapella {
   repeated HistoricalSummary historical_summaries = 11003 [(ethereum.eth.ext.ssz_max) = "16777216"]; // [New in Capella]
 }
 
+message BeaconStateDeneb {
+  // Versioning [1001-2000]
+  uint64 genesis_time = 1001;
+  bytes genesis_validators_root = 1002 [(ethereum.eth.ext.ssz_size) = "32"];
+  uint64 slot = 1003 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.Slot"];
+  Fork fork = 1004;
```

Comment:
Why do we need a different `BeaconStateDeneb`?

Reply:
payload header is different